openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1655,6 +1655,7 @@ static int cert_requested(SSL* ssl, X509** x509Out, EVP_PKEY** pkeyOut) {
             break;
     }
 #else
+    (void) ssl2_ctype;
```

Comment:
I assumed that all declarations had to be at the beginning of the block (c89 I think?)  If that isn't a requirement I can move it down.